app-emulation/qemu/qemu-9999.ebuild

```diff
@@ -27,7 +27,9 @@ if [[ ${PV} == *9999* ]]; then
 	EGIT_REPO_URI="https://gitlab.com/qemu-project/qemu.git/"
 	EGIT_SUBMODULES=()
 	inherit git-r3
-	SRC_URI=""
+	SRC_URI="https://gitlab.com/qemu-project/keycodemapdb/-/archive/f5772a62ec52591ff6870b7e8ef32482371f22c6/keycodemapdb-f5772a62ec52591ff6870b7e8ef32482371f22c6.tar.gz -> keycodemapdb.tar.gz
+		https://gitlab.com/qemu-project/berkeley-softfloat-3/-/archive/b64af41c3276f97f0e181920400ee056b9c88037/berkeley-softfloat-3-b64af41c3276f97f0e181920400ee056b9c88037.tar.gz -> berkeley-softfloat-3.tar.gz
+		https://gitlab.com/qemu-project/berkeley-testfloat-3/-/archive/40619cbb3bf32872df8c53cc457039229428a263/berkeley-testfloat-3-40619cbb3bf32872df8c53cc457039229428a263.tar.gz -> berkeley-testfloat-3.tar.gz"
```

Comment:
Can you separate the commit hashes into their own variables so we don't have to duplicate them a bunch of places? Something like this, towards the top of the ebuild:

```
KEYCODEMAPDB_COMMIT=...
BERKELEY_SOFTFLOAT_COMMIT=...
BERKELEY_TESTFLOAT_COMMIT=...
```

Maybe those names are a bit long I don't have a strong opinion about the naming.

Reply:
Good point. Let me update that.